### PR TITLE
net: add page

### PR DESCRIPTION
**Pull request by FallenGameR**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/windows/net.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# net
+
+> System utility to view and modify network-related settings.
+> More information: <https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/gg651155(v=ws.11)>.
+
+- See the help for a subcommand or list all the subcommands:
+
+`net help {{subcommand_or_empty}}`
+
+- Start or stop a windows service syncronously:
```
Comment (kbdharun) [inline pages/windows/net.md:10]:
```suggestion
- Start or stop a Windows service synchronously:
```

File: pages/windows/net.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`net {{start|stop}} {{service_name}}`
+
+- Make sure an SMB share is avaialable in the current console:
```
Comment (kbdharun) [inline pages/windows/net.md:14]:
```suggestion
- Make sure an SMB share is available in the current console:
```

File: pages/windows/net.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`net session`
+
+- Show users in a local security group:
+
+`net localgroup "{{Administrators}}"`
+
+- Add a user to the local security group (run in elevated console):
+
+`net localgroup "{{Administrators}}" {{user_name}} /add`
```
Comment (kbdharun) [inline pages/windows/net.md:32]:
```suggestion
`net localgroup "{{Administrators}}" {{username}} /add`

- Display help for a subcommand:

`net help {{subcommand}}`

- Display help:

`net help`
```

File: pages/windows/net.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# net
+
+> System utility to view and modify network-related settings.
+> More information: <https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/gg651155(v=ws.11)>.
+
+- See the help for a subcommand or list all the subcommands:
+
+`net help {{subcommand_or_empty}}`
+
```
Comment (kbdharun) [inline pages/windows/net.md:9]:
```suggestion
```

File: pages/windows/net.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+- Start or stop a windows service syncronously:
+
+`net {{start|stop}} {{service_name}}`
+
+- Make sure an SMB share is avaialable in the current console:
+
+`net use {{\\smb_shared_folder}} /USER:{{user_name}}`
```
Comment (kbdharun) [inline pages/windows/net.md:16]:
```suggestion
`net use {{\\smb_shared_folder}} /USER:{{username}}`
```

File: pages/windows/net.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+> System utility to view and modify network-related settings.
+> More information: <https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/gg651155(v=ws.11)>.
+
+- See the help for a subcommand or list all the subcommands:
+
+`net help {{subcommand_or_empty}}`
+
+- Start or stop a windows service syncronously:
+
+`net {{start|stop}} {{service_name}}`
```
Comment (kbdharun) [inline pages/windows/net.md:12]:
```suggestion
`net {{start|stop}} {{service}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**